ballerina/build.gradle

```diff
@@ -1,6 +1,5 @@
-
 plugins {
-    id 'io.ballerina.plugin'
+    id "io.ballerina.plugin" version "${ballerinaGradlePluginVersion}"
```

Comment:
Is this change related to workflows ? Best practice is to have one PR for a specific change.

Reply:
What I should do when I working on one thing and I found out another small mistake that is not related to issue that I am working on, shouldn't I fix that small mistake? What should I do?

Reply:
This kind of things happen. But its best if we could send another PR for the fix (using another branch) and merge it to master before completing the feature you are working on.